src/rbd.cc

```diff
@@ -499,6 +475,7 @@ static int do_clone(librbd::RBD &rbd, librados::IoCtx &p_ioctx,
   if ((features & RBD_FEATURE_LAYERING) != RBD_FEATURE_LAYERING) {
     return -EINVAL;
   }
+  features &= ~RBD_FEATURE_STRIPINGV2;
 
   return rbd.clone(p_ioctx, p_name, p_snapname, c_ioctx, c_name, features,
```

Comment:
we should really use clone2() here, and pass in the striping info the user may have specified